src/main/java/com/epam/aidial/cfg/service/ToolSetResourceService.java

```diff
@@ -41,7 +44,7 @@
 @LogExecution
 public class ToolSetResourceService implements ResourceService {
     private static final String BASE_PATH = "public/";
-
+    private static final String SIGN_IN_MESSAGE = "For authenticationType %s, the field %s must be provided.";
```

Comment:
Let's change to "The '%s' field must have a value. Authentication type: '%s'"

Reply:
done